### PR TITLE
fix(tools): Handle code block containing hasOwnProperty

### PR DESCRIPTION
**Pull request by RandellDawson**
This PR fixes an edge case for the `Validate Inline Code Blocks` custom QA check, where by chance one of the code blocks contains the string "hasOwnProperty".   I added a new test just for this edge case also.

You can run `npm run test` and everything should pass.

### CODE REVIEW

File: validate-inline-code-blocks.js [hunk 1]
```diff
@@ -10,9 +10,10 @@ function createCodeArr(arr) {
 
 function countElements(arr) {
   var obj = {};
+  obj.__tempHasOwnProperty = Object.hasOwnProperty;
```
Comment (nhcarrigan) [inline validate-inline-code-blocks.js:13]:
```suggestion
  obj.__tempHasOwnProperty = Object.prototype.hasOwnProperty;
```

**Review by nhcarrigan — APPROVED** [review]
